eng/InternalTools.props

```diff
@@ -3,7 +3,16 @@
 <Project>
   <Import Project="dependencies.props" />
 
-  <ItemGroup>
+  <PropertyGroup>
+    <UsingILCTools Condition="$(TargetGroup.EndsWith('aot'))">true</UsingILCTools>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetGroup)' != 'netfx' AND '$(OS)' == 'Windows_NT'">
```

Comment:
@ericstj maybe this is better here than in Versions.props?

Reply:
Shouldn't this check on TargetOS instead of OS?

Reply:
I'm not sure TargetOS will be defined in this project because we don't import Directory.Build.props at all. So that is why I used the OS msbuild property

Reply:
right but you could also flow that into the project, right? More important, how are we going to flow these properties into this call: https://github.com/dotnet/corefx/blob/master/eng/Tools.props#L72? I guess also with an env var?

Reply:
That is already flown as part of this PR: https://github.com/dotnet/corefx/pull/35433/files#diff-c9ed7fc6ec49880569e6c2eb08120fc4R72

TargetGroup is available for Tools.props since that is called from arcade, and we flow that property through our build scripts. The reason why I opted for OS rather than TargetOS here, was to not force people locally to flow that property when building.

Reply:
Please just remove this entirely.  I'll set things up in my PR.

Reply:
Fixed, I just removed it :smile: